src/Plaid/PlaidOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Going.Plaid
+{
+	/// <summary>
+	/// Implements the IOptions pattern for reading a 'Plaid' section from an IConfiguration object.
+	/// </summary>
+	public class PlaidOptions
+	{
+		/// <summary>
+		/// The section name in the IConfiguration that contains PlaidOptions settings
+		/// </summary>
+		public const string SectionKey = "Plaid";
+
+		/// <summary>
+		/// client_id is the same across all environments
+		/// !!! THIS SHOULD NEVER BE SHARED IN CLIENT-SIDE CODE !!!
+		/// See: https://plaid.com/docs/quickstart/#api-keys
+		/// </summary>
+		public string? ClientId { get; set; } = null;
+
+		/// <summary>
+		/// you have a unique secret for each API environment
+		/// !!! THIS SHOULD NEVER BE SHARED IN CLIENT-SIDE CODE !!!
+		/// See: https://plaid.com/docs/quickstart/#api-keys
+		/// </summary>
+		public string? Secret { get; set; } = null;
+
+		/// <summary>
+		/// OPTIONAL: Use to specify a default AccessToken for ALL requests that do not provide their own AccessToken 
+		/// NOTE: If you are acting as a User in a session (e.g. on a Web API), you probably don't want to set this system-wide. 
+		///       It would be safer to use one <see cref="PlaidClient"/> for system-level operations, and a different client instance 
+		///       to act as the User to ensure that the default access token is never attached on User-specific requests.
+		/// </summary>
+		public string? DefaultAccessToken
+		{
+			get => _defaultAccessToken;
+			set => _defaultAccessToken = string.IsNullOrWhiteSpace(value) ? null : value;
+		}
+		private string? _defaultAccessToken;
+
+		/// <summary>
+		/// Sandbox | Development | Production
+		/// See: https://plaid.com/docs/quickstart/#api-enviroments
+		/// </summary>
+		public string Environment { get; set; } = Plaid.Environment.Sandbox.ToString();
```

Comment:
Preferred to set the type as `Plaid.Environment`. Is this possible with `IOptions<T>`?

Reply:
oops. I meant to go back and clean loop that up. I think so - will change it.